widgets/contact/styles/default.less

```diff
@@ -126,7 +156,7 @@
 		background: @error_background;
 		border: 1px solid @error_border;
 		color: @error_text;
-		border-radius: 2px;
+		.rounded(2px);
```

Comment:
There's very little reason to do this, but I was doing it for consistency reasons. It's fine though.